ivy.el

```diff
@@ -2417,6 +2418,21 @@ Insert .* between each char."
         (setq ivy--subexps (length (match-string 2 str))))
     str))
 
+(defun ivy--subseq-fuzzy (str)
+  "Build a regex matching candidates containing a subsequence of STR.
+
+Space in STR is ignored.
+
+This function filters out candidates of which STR is not a subsequence.
+Technically this is not required, because the O(N*M) subsequent sequence
+alignment `ivy--fuzzy-sort' will give an awful score for such candidates,
+though slower."
+  (mapconcat
+   (lambda (x)
+     (format "\\(%s\\)" (regexp-quote (string x))))
+   (delete ?\  (string-to-list str))
```

Comment:
Please write `?\s` instead of `?\ `. You can also use `delq` instead of `delete` here.

Reply:
You can also use `char-to-string` instead of `string` for the tiniest of speedups.